capitolwords_ng/capitolweb/cwapi/views.py

```diff
@@ -111,3 +116,48 @@ def search_by_params(request):
         return JsonResponse(response.to_dict())
     return JsonResponse("Found nothing")
 
+@api_view(['GET'])
+def count_of_term_in_content(request, term):
+    """
+    Count the mention of a term in content
+
+    time_period(optional, default to 'month') - period of time to go
+                                                back from today
+                                                month | week | day
+
+    Example:
+        http://127.0.0.1:8000/cwapi/count/obamacare/?time_period=week
+        http://127.0.0.1:8000/cwapi/count/obamacare/
+
+    :param request:
+    :param term: term whose occurances we are counting
+    :return: number of occurances of the term in content
+    """
+    time_period = 'month'
```

Comment:
You could shorten this and the next couple of lines to this:
```python
time_period = request.query_params.get('time-period', 'month')
```
I'm not super familiar with django, but @rmangi or @heavi5ide may know if there is a way to set default arguments for a route.